engine/runtime/src/main/java/org/enso/interpreter/runtime/Context.java

```diff
@@ -482,9 +482,9 @@ public Optional<AtomConstructor> getDateConstructor() {
       ensureModuleIsLoaded(stdDateModuleName);
       Optional<Module> dateModule = findModule(stdDateModuleName);
       if (dateModule.isPresent()) {
-        date =
-            Optional.ofNullable(
-                dateModule.get().getScope().getConstructors().get(stdDateConstructorName));
+        final Module m = dateModule.get();
```

Comment:
Don't believe this used any more?

Reply:
Good catch! Yes, I believe this method can be removed: 4eb0a243d12be3ae7ee196cb358da32f9fe4d109